src/App.js

```diff
@@ -1,19 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 
 import AddView from './components/AddView';
 import ListView from './components/ListView';
 import Navigation from './components/Navigation';
-import Counter from './components/Counter';
+import Home from './components/Home';
+
+import getToken from './lib/tokens';
 import './App.css';
 
 const App = () => {
+  const [token, setToken] = useState(localStorage.getItem('token' || ''));
+
+  const handleTokenCreation = () => {
+    setToken(getToken());
+  };
   return (
     <BrowserRouter>
       <Switch>
-        <Route path="/add-view" component={AddView} exact />
-        <Route path="/list-view" component={ListView} exact />
-        <Counter />
+        <Route exact path="/">
+          <Home token={token} handleTokenCreation={handleTokenCreation} />
+        </Route>
+        <Route path="/add-view">
+          <AddView token={token} />
+        </Route>
+        <Route path="/list-view">
+          <ListView token={token} />
```

Comment:
Are you going to use these props somewhere in these files in your branch? If not, it is kind of unnecessary to write code that is not used, I'd say.

Reply:
hi @YuriDevAT , the token prop in every component has been added only because we knew the other team would need to use it, but I'll delete it now. Thank you.

Reply:
hi @YuriDevAT, the token prop has been removed.